Samples/configure_cloud.ps1

```diff
@@ -18,9 +18,6 @@ Enter your Service DNS name (ex: contoso.cloudapp.net).
 .PARAMETER CName
 Enter your Service CName (ex: contoso.net).
 
-.PARAMETER CertThumbprint
```

Comment:
Is the `configure_cloud.ps1` still valid? Are we using it in the steps above to configure the files? If not we can clean it up?

Reply:
Sync'd and will keep the change.